man/systemd-detect-virt.xml

```diff
@@ -143,11 +143,16 @@
             <entry>Parallels Desktop, Parallels Server</entry>
           </row>
 
-           <row>
+          <row>
             <entry><varname>bhyve</varname></entry>
             <entry>bhyve, FreeBSD hypervisor</entry>
           </row>
 
+          <row>
+            <entry><varname>qnx</varname></entry>
+            <entry>QNX hypervisor</entry>
+          </row>
```

Comment:
hmm, could you also update systemd.unit(5) please, there's a brief list of supervisor detected in the description of `ConditionVirtualization=`

Reply:
Thanks, done.